### PR TITLE
addedScriptz

### PR DESCRIPTION
**Pull request by ankitpar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by ankitpar** [comment]
Build getting killed by other script retriggering

File: g/protobuf/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (prashantkhoje) [inline g/protobuf/LICENSE:1]: Please move protobuf under folder 'p'

File: g/protobuf/protobuf_ubi8_rhel.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+
```
Comment (prashantkhoje) [inline g/protobuf/protobuf_ubi8_rhel.sh:3]: Please remove extra blank line(s) from the file.

File: g/protobuf/protobuf_ubi8_rhel.sh [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+# Variables
+
+PACKAGE_NAME=protobuf
+PACKAGE_URL=https://github.com/protocolbuffers/protobuf.git
+#PACKAGE_VERSION is configurable can be passed as an argument.
```
Comment (prashantkhoje) [inline g/protobuf/protobuf_ubi8_rhel.sh:26]: This script doesn't support setting PACKAGE_VERSION from passed argument.

File: g/protobuf/protobuf_ubi8_rhel.sh [hunk 4]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+
+# ----------------------------------------------------------------------------
+# Package          : protobuf
+# Version          : v21.1
+# Source repo      : https://github.com/protocolbuffers/protobuf.git
+# Tested on        : UBI 8.5
+# Language         : C++
+# Travis-Check     : True
```
Comment (prashantkhoje) [inline g/protobuf/protobuf_ubi8_rhel.sh:10]: Please fix travis build failure.

**Comment by ankitpar** [comment]
Created a new directory structure as per review comment

**Pull request closed without merging**